<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Sangee_28" failures="0" tests="11" name="com.test.M10_RegisterPage" time="462.417" errors="1" timestamp="2024-10-24T12:48:16 IST" skipped="0">
  <testcase classname="com.test.M10_RegisterPage" name="NameValidation" time="46.793"/>
  <system-out/>
  <testcase classname="com.test.M10_RegisterPage" name="NameValidation" time="45.469"/>
  <system-out/>
  <testcase classname="com.test.M10_RegisterPage" name="NameValidation" time="45.810"/>
  <system-out/>
  <testcase classname="com.test.M10_RegisterPage" name="EmailValidation" time="45.770"/>
  <system-out/>
  <testcase classname="com.test.M10_RegisterPage" name="EmailValidation" time="45.223"/>
  <system-out/>
  <testcase classname="com.test.M10_RegisterPage" name="mobileNumberValidation" time="0.000">
    <error message="
Method private void com.test.M10_RegisterPage.mobileNumberValidation(java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.lang.InterruptedException requires a @DataProvider named : mobilenum" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Method private void com.test.M10_RegisterPage.mobileNumberValidation(java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.lang.InterruptedException requires a @DataProvider named : mobilenum
at org.testng.internal.Parameters.findDataProvider(Parameters.java:561)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- mobileNumberValidation -->
  <system-out/>
  <testcase classname="com.test.M10_RegisterPage" name="PasswordValidation" time="45.543"/>
  <system-out/>
  <testcase classname="com.test.M10_RegisterPage" name="PasswordValidation" time="45.291"/>
  <system-out/>
  <testcase classname="com.test.M10_RegisterPage" name="ValidInformation" time="47.611"/>
  <system-out/>
  <testcase classname="com.test.M10_RegisterPage" name="ValidInformation" time="47.679"/>
  <system-out/>
  <testcase classname="com.test.M10_RegisterPage" name="ValidInformation" time="47.228"/>
  <system-out/>
</testsuite> <!-- com.test.M10_RegisterPage -->
